src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table-actions/table-actions.component.ts

```diff
@@ -35,6 +35,13 @@ export class TableActionsComponent implements OnInit {
     this.updateDropDownActions();
   }
 
+  public toClassName(name: string): string {
```

Comment:
Cool extension :+1: 
But methods are public by default:
```suggestion
  toClassName(name: string): string {
```